src/ldp/operations/PatchOperationHandler.ts

```diff
@@ -21,6 +29,12 @@ export class PatchOperationHandler extends OperationHandler {
   }
 
   public async handle(input: Operation): Promise<ResponseDescription> {
+    // Solid, §2.1: "A Solid server MUST reject PUT, POST and PATCH requests
+    // without the Content-Type header with a status code of 400."
+    if (!input.body?.metadata.contentType) {
+      this.logger.warn('PATCH operations require a content-type');
+      throw new BadRequestHttpError('PATCH operations require a content-type');
+    }
```

Comment:
Check if our body parsers do this. I would move the contract there.
Then again, a double check does not hurt.

Also, with error messages, try suggesting a solution (perhaps a bit more instructive):
> No Content-Type header specified on PATCH request